app/models/vm_migrate_workflow.rb

```diff
@@ -38,6 +40,15 @@ def get_source_and_targets(refresh = false)
     @target_resource = result
   end
 
+  def add_target(dialog_key, key, klass, result)
+    super unless field_unsupported(key)
+  end
+
+  def field_unsupported(key)
```

Comment:
@mkanoor can you have a look at this or somebody from your team that has experience with this?
I'm not sure if this is how we handle partially supported fields